### PR TITLE
`[stable/charts/prometheus]`Add ability to add array of snippets with your init containers for promtheus server

### PR DESCRIPTION
**Pull request by msharbaji**
#### What this PR does / why we need it:
This PR to add array of snippets with your init containers for promtheus server.
Why: 
Specialized containers that run before app containers in a Pod . Init containers can contain utilities or setup scripts not present in an app image. 
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by msharbaji** [comment]
@mgoodness @gianrubio Can you please review my PR to add ability to add snippets for the init containers.

File: stable/prometheus/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: prometheus
-version: 8.15.0
+version: 8.15.1
```
Comment (Aisuko) [inline stable/prometheus/Chart.yaml]: Bump the MINOR version could make sense. `8.xx.0`

**Review by Aisuko — CHANGES_REQUESTED** [review]
Please resolve the confliction. And there may some a piece of advice.

File: stable/prometheus/Chart.yaml (hunk 1 above)
Comment (msharbaji) [reply stable/prometheus/Chart.yaml]: @Aisuko I have resolved the conflict and bumped the minor version, please review :)

**Comment by Aisuko** [comment]
cc @gianrubio Hello sir, any feedback would be good.

**Comment by msharbaji** [comment]
> cc @gianrubio Hello sir, any feedback would be good.

Is there anything I need to do going forward?

**Comment by Aisuko** [comment]
We need to wait for the chart's OWNER to confirm this feature is useful and merge it. I would post this PR into our channel on Slack. Thanks for your PR.

**Pull request closed without merging**